### ISSUE TITLE
What do these warnings mean?

### ISSUE THREAD

**Issue by jellis18** [nanograv/PINT#317]
I've been incorporating `PINT` into `enterprise` and I keep getting these warnings. What do these warnings mean?
```python
WARNING: Cut off delay component 'BinaryELL1' is not included in delay calculation. [pint.models.timing_model]
WARNING: astropy: Cut off delay component 'BinaryELL1' is not included in delay calculation.
```

**Comment by luojing1211** [comment]
Hi Justin,
It is a debug warning. I am going to fix it.

-Jing Luo

**Comment by luojing1211** [comment]
This problem has been resolved by pull request #318 .
